softwareComponents/configuration/include/configuration/joints.hpp

```diff
@@ -14,6 +14,8 @@
 
 namespace rofi::configuration {
 
+using namespace rofi::configuration::matrices;
```

Comment:
This is polluting namespace `rofi::configuration`. If you want to have it in this namespace, remove the `matrices` namespace.

I think that using it in a `cpp` file is fine, but not in a header file.

Reply:
Should be fixed by now. Now it uses only the two types; `Matrix` and `Vector`.